src/presentation/http/router/noteSettings.test.ts

```diff
@@ -202,35 +306,59 @@ describe('NoteSettings API', () => {
 
   describe('PATCH /note-settings/:notePublicId ', () => {
     test('Update note settings by public id with 200 status, user is creator of the note', async () => {
-      const userId = 2;
-      const accessToken = global.auth(userId);
+      /**
+       * truncate all tables, which are needed
+       * restart autoincrement sequences for data to start with id 1
+       *
+       * TODO get rid of restarting database data in tests (move to beforeEach)
+       */
+      await global.db.truncateTables();
+
+      /** create test user */
+      const user = await global.db.insertUser({
+        email: 'a@a.com',
+        name: 'Test user 1',
+      });
 
-      const userNote = notes.find(newNote => {
-        return newNote.creator_id === userId;
+      /** create test note for created user */
+      const note = await global.db.insertNote({
+        creatorId: user.id,
+        publicId: 'Pq1T9vc23Q',
       });
 
-      const updatedNoteSettings = {
-        'id': 54,
-        'noteId': 54,
-        'customHostname': 'codex.so',
-        'isPublic': false,
-        'invitationHash': 'FfAwyaR80C',
-      };
+      /** create test note settings for created note */
+      const noteSettings = await global.db.insertNoteSetting({
+        noteId: note.id,
+        isPublic: true,
+      });
+
+      /** create test team member for created note */
+      await global.db.insertNoteTeam({
+        userId: user.id,
+        noteId: note.id,
+        role: 0,
+      });
```

Comment:
creator should not be included in a team. Do you think teams is required of this test?